docs/source/guides/user/chapters/results.rst

```diff
@@ -132,9 +132,10 @@ On GNU/Linux systems, a child process should be *"an exact duplicate
 of the parent process, except"* some items that are documented in
 the ``fork(2)`` man page.
 
-.. note:: The next Runner (``--test-runner='nrunner'``) has support to
-   different spawners types (podman, process, etc..).  For more information,
-   visit the ``nrunner.spawner`` configuration option.
+.. note:: the non-legacy runner (once known as the N(ext) Runner, AKA
```

Comment:
I found it a bit confusing calling the runner `non-legacy`. What if, as the commit message infers, we call `current runner`?

Reply:
OK, I'll change that and force push.